Fix SVGs

seanlynchwv opened the pull request:
### Description

Adding custom SVGs now works

Fixed by https://github.com/PHPOffice/PHPPresentation/issues/732#issuecomment-1453421226 

Fixes # (issue)

https://github.com/PHPOffice/PHPPresentation/issues/732

### Checklist:

- [x] My CI is :green_circle:
- [x] I have covered by unit tests my new code (check build/coverage for coverage report)
- [ ] I have updated the [documentation](https://github.com/PHPOffice/PHPPresentation/tree/develop/docs) to describe the changes
- [x] I have updated the [changelog](https://github.com/PHPOffice/PHPPresentation/blob/develop/docs/changes/1.1.0.md)

seanlynchwv committed:
fix svgs

seanlynchwv committed:
update changelog 1.2.0

seanlynchwv commented:
Not sure why this is failing now, was passing earlier

Progi1984 commented on docs/changes/1.2.0.md:
```diff
@@ -30,6 +30,7 @@
 - ODPresentation Reader : Read differents units for margin by [@Progi1984](https://github.com/Progi1984) fixing [#830](https://github.com/PHPOffice/PHPPresentation/pull/830) in [#847](https://github.com/PHPOffice/PHPPresentation/pull/847)
 - PowerPoint2007 Reader : Fixed loading of fonts by [@ag3202](https://github.com/ag3202) and [@Progi1984](https://github.com/Progi1984) in [#851](https://github.com/PHPOffice/PHPPresentation/pull/851)
 - PhpOffice\PhpPresentation\Style\TextStyle : Fixed typo for default indent by [@Progi1984](https://github.com/Progi1984) in [#857](https://github.com/PHPOffice/PHPPresentation/pull/857)
+- Fixed adding custom SVGs by [@seanlynchwv](http://github.com/seanlynchwv) in [#881](https://github.com/PHPOffice/PHPPresentation/pull/881)
```
Could you move to 1.3.0.md ?

Progi1984 reviewed (changes requested):
@seanlynchwv Could you add an unit test please ?

seanlynchwv committed:
Merge remote-tracking branch 'upstream/master' into fix/svg-error

seanlynchwv committed:
update changelog

seanlynchwv committed:
add new test

seanlynchwv committed:
Merge branch 'master' into fix/svg-error

seanlynchwv committed:
Fix PHP CS Fixer violations in ContentTypesTest

Orders imports correctly, imports ZipArchive instead of referencing it
with a leading backslash, and uses self::assert* per this repo's
php_unit_test_case_static_method_calls rule.

seanlynchwv committed:
Fix flaky ObjectsChartTest::testTypeAxisUnit

mt_rand(0, 100) occasionally produced 0, which the OpenDocument 1.2
schema rejects for chart:interval-minor-divisor/chart:interval-major,
failing RelaxNG validation with a confusing unrelated-looking error.
Use mt_rand(1, 100) instead.

seanlynchwv committed:
Merge branch 'master' into fix/svg-error

seanlynchwv committed:
Update src/PhpPresentation/Writer/PowerPoint2007/ContentTypes.php

Co-authored-by: Progi1984 <progi1984@gmail.com>

seanlynchwv committed:
Use master's max(1, mt_rand(0, 100)) form for testTypeAxisUnit

seanlynchwv committed:
Remove duplicate blank line in ContentTypes writer
